### ISSUE TITLE
[Old Hillsbrad Foothills] Thrall script aborts when stealthed rogue or druid close to Captain Skarloc

### ISSUE THREAD

**Issue by balleny** [chromiecraft/chromiecraft#5972]
### What client do you play on?

enUS

### Faction

Both

### Content Phase:

Generic

### Current Behaviour

During the part of the segment where Captain Skarloc arrives and is engaged by Thrall, the script errors out and stops, if a rogue or druid in stealth is close to Captain Skarloc.
it seems that Captain Skarlocs pathing is interrupted.

### Expected Blizzlike Behaviour

the RP part of Captain Skarloc arriving and engaging Thrall shouldnt be influenced by stealthed units close by

### Source

.

### Steps to reproduce the problem

.

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/30197f40d687eae97a07d6730b6933c63f1a3983

### Operating system

Ubuntu 20.04

### Modules

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-bg-item-reward](https://github.com/azerothcore/mod-bg-item-reward)
- [mod-BreakingNewsOverride](https://github.com/AnchyDev/BreakingNewsOverride)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chat-transmitter](https://github.com/azerothcore/mod-chat-transmitter)
- [mod-cta-switch](https://github.com/azerothcore/mod-cta-switch)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna](https://github.com/azerothcore/mod-eluna)
- [mod-instanced-worldbosses](https://github.com/nyeriah/mod-instanced-worldbosses)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-low-level-rbg](https://github.com/azerothcore/mod-low-level-rbg)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-progression-system](https://github.com/azerothcore/mod-progression-system)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-rdf-expansion](https://github.com/azerothcore/mod-rdf-expansion)
- [mod-transmog](https://github.com/azerothcore/mod-transmog)
- [mod-weekend-xp](https://github.com/azerothcore/mod-weekend-xp)
- [mod-zone-difficulty](https://github.com/azerothcore/mod-zone-difficulty)
- [lua-carbon-copy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-event-scripts](https://github.com/55Honey/Acore_eventScripts)
- [lua-exchange-npc](https://github.com/55Honey/Acore_ExchangeNpc)
- [lua-level-up-reward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-recruit-a-friend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-temp-announcements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-zonecheck](https://github.com/55Honey/acore_Zonecheck)

### Customizations

None

### Server

ChromieCraft

**Comment by unitscan** [comment]
I can confirm this.

**Comment by kerricsson** [comment]
Still valid at AzerothCore rev. 4c287e2873c3+ 2023-08-22 12:13:36 +0200 (master branch) (Unix, RelWithDebInfo, Static). Very annoying in LFG since significant numbers of roques/cats queue as dps and this problem happens almost every day and can lead to significant issues - deserter buff and minus 2 badges (since HC can not be completed) to lost daily quest and over HC cd.

**Comment by balleny** [comment]
workaround is to pull mobs to thrall and then vanish / stealth etc.
they kill thrall and reset the borked script

**Comment by kerricsson** [comment]
Yes, I mentioned it at mine report, but it's deffo not like in should be and it can lead to massive group disband/kicks that ruins player experience.

**Comment by kerricsson** [comment]
Good day, Just want to know, can this issue be kinda prioritized after Kara finalization? Because it still cause big problems even for experienced groups every day. For example, speaking about OHF - yesterday it caused party disband again and due to new ppl coming, they don't know about this bug and cause full script stopped. Also, sometimes it's not even possible to prevent ppl due to lack of LATAM languages knowledge or lack of Chinese knowledge. Thanks in advance for understanding.

**Comment by Plan414** [comment]
This bug is still an issue with unknowing stealth players. The only option after is to disband the group. Can we prioritise this fix?
